.github/workflows/push_checking.yml

```diff
@@ -18,18 +18,11 @@ jobs:
           node-version: ${{ matrix.node-version }}
 
       - name: Install dependencies
-
-        run: yarn install --frozen-lockfile
```

Comment:
Hey, I was going over this and please never never do `npm install` unless you want to get unexpected results.

The lock file (package-lock.json or yarn.lock) has a purpose. And it is that everybody that _installs_ the dependencies gets exactly the same versions. That's what `npm ci` will do. It will install exactly the same versions listed in the lock file. On the other hand, `npm install` will get the latest packages for the direct dependencies and sub-dependencies that meet the requirements. Hence, everybody will have different versions, including the package that was built and released.

The problem arises when due to mismatching versions, someone does not have a bug, while other developer or user has.

Reply:
the best solution in my opinion would be to keep the `package-lock.json` but to replace `npm i` by `npm ci`